File: c/PROBLEMS/1043.c
#include <stdio.h>

int main(){
    double a, b, c;

    scanf ("%lf %lf %lf", &a, &b, &c);

    if (a+b > c && a+c > b && b+c > a){
        double perimetro = a + b + c;
        printf("Perimetro = %.1f\n", perimetro);
    }
    else{
        double area = ((a + b) * c) / 2;
        printf("Area = %.1f\n", area);
    }
}

File: js/PROBLEMS/1043.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8')
var lines = input.split(' ')

let a = parseFloat(lines[0])
let b = parseFloat(lines[1])
let c = parseFloat(lines[2])

if (a+b > c && a+c > b && b+c > a){
    let perimetro = a + b + c;
    console.log(`Perimetro = ${perimetro.toFixed(1)}`);
}
else{
    let area = ((a + b) * c) / 2;
    console.log(`Area = ${area.toFixed(1)}`);
}